python/looker_sdk/rtl/api_methods.py

```diff
@@ -44,7 +44,7 @@
         Sequence[model.Model],
     ]
 ]
-TStructure = Optional[Union[Type[str], serialize.TStructure]]
+TStructure = Optional[Union[Any, Type[str], serialize.TStructure]]
```

Comment:
the new `api_spec` method returns `Any` so needed to add that to the union type here

Reply:
Yeah, it's now (correctly) a JSON document